app/repositories/BulkCalculationRepository.scala

```diff
@@ -452,7 +452,7 @@ class BulkCalculationMongoRepository @Inject()(metrics: ApplicationMetrics)(impl
           }
 
           val insertResult = proxyCollection.insert(strippedBulk).flatMap {_ =>
-            proxyCollection.insert[ProcessReadyCalculationRequest](ordered = false).many(bulkDocs)
```

Comment:
why remove the type annotation?

Reply:
Type put back in now

Reply:
Not what I meant. The type on the insert (as it was before)

Reply:
done now